app/src/renderer/components/staking/LiValidator.vue

```diff
@@ -1,28 +1,28 @@
 <template lang='pug'>
-.li-validator(:class='styles'): .li-validator__values
-  .li-validator__value.name
-    router-link(:to="{ name: 'validator', params: { validator: validator.id }}")
-      span.validator-profile__status(v-tooltip.top="status")
+li.li-validator(:class='styles')
+  router-link(:to="{ name: 'validator', params: { validator: validator.id }}")
+    .li-validator__value.name
+      span.validator-profile__status(v-bind:class="statusColor" v-tooltip.top="status")
       img.avatar(v-if="validator.keybase" :src="validator.keybase.avatarUrl" width="48" height="48")
       img.avatar(v-else src="~assets/images/validator-icon.svg" width="48" height="48")
       .vert
         .top {{ validator.description.moniker }}
         .bottom {{ shortAddress(validator.id)}}
-  .li-validator__value.your-votes
-    span {{ yourVotes }}
-  .li-validator__value.your-rewards
-    span n/a
-  .li-validator__break: span
-  .li-validator__value.percent_of_vote
-    span {{ validator.percent_of_vote ? validator.percent_of_vote : `n/a` }}
-  .li-validator__value.uptime
-    // add .green .yellow or .red class to this span to trigger inidication by color
-    span {{ uptime }}
-  .li-validator__value.commission
-    span {{ commission }}
-  .li-validator__value.slashes
-    // add .green .yellow or .red class to this span to trigger inidication by color
-    span {{ slashes }}
+    .li-validator__value.your-votes
+      span {{ yourVotes }}
```

Comment:
@faboweb we should replicate the same condition being used in `PageValidator.vue` with the `myBonds()` function to show `< 0.01` when the bond amount is low